test/format/checksum.c

```diff
@@ -0,0 +1,136 @@
+#include "format.h"
+
+#define FNV_PRIME 0x00000100000001b3
+
+struct checksum_table_arg {
+    WT_SESSION *session;
+    uint64_t *hash;
```

Comment:
Do we need to make it pointer here?

```suggestion
    uint64_t hash;
```

Reply:
Nope -- changed in `255f2dae31`.